js/script.js

```diff
@@ -5,3 +5,15 @@ menu_btn.addEventListener('click', function () {
   menu_btn.classList.toggle('is-active');
   mobile_menu.classList.toggle('is-active');
 });
+
+const modal = document.getElementById('modal');
```

Comment:
- [ ] The JavaScript code should implement an array of objects to store JavaScript objects containing the information for each project instead of using HTML code. This helps in organizing data into objects and can be used for later purposes.
- [ ] Each object should contain at least the following information about the project: name, description, featuring-image, technologies, link to live version, and link to the project source code.
- [ ] Modal popup data should be populated with the data from the projects array.

 **Please have another look at the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md).**